logstash-core/spec/logstash/pipeline_spec.rb

```diff
@@ -371,20 +371,6 @@ def flush(options)
       eos
     }
 
-    let(:test_config_with_output_workers) {
```

Comment:
It's not clear to me why this config has been removed

Reply:
i'll add a note in the commit message, this is a leftover from the time when we had output workers, but the let isn't used anywhere